my-app/src/backend/controllers/CartController.js

```diff
@@ -119,8 +119,11 @@ export const updateCartItemHandler = function (schema, request) {
     }
     const userCart = schema.users.findBy({ _id: userId }).cart;
     const { action } = JSON.parse(request.requestBody);
+    
     if (action.type === "increment") {
+      console.log("From controller",action)
```

Comment:
In case you forgot to remove logs from your file, just remove it before creating a PR.